test/style/module_directives/alias_lifting_test.exs

```diff
@@ -179,6 +179,19 @@ defmodule Styler.Style.ModuleDirectives.AliasLiftingTest do
   end
 
   describe "it doesn't lift" do
+    test "collisions with configured modules" do
+      Styler.Config.set!(alias_lifting_exclude: ~w(C)a)
```

Comment:
@GregMefford lmk if you have an idea for clean config setting per test. i had it built into assert_style and honestly could still do that. maybe smart to just add a `set_config` in style_case that does the `on_exit(fn -> set!([]) end)`...